pkg/auth/authenticator/password/keystonepassword/keystonepassword.go

```diff
@@ -0,0 +1,66 @@
+package keystonepassword
+
+import (
+	"fmt"
+	"errors"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack"
```

Comment:
@liggitt @deads2k Looks like the gophercloud deps are already here as of this commit: https://github.com/openshift/origin/commit/83f9bfd2d4e5ecabfbd46c67f78edda058656051